script/sync-translation

```diff
@@ -1,59 +0,0 @@
-#!/bin/sh
```

Comment:
I'd say some script for checking whether the translation is up-to-date is still needed.

Reply:
@webknjaz There will be no need to have this synced to separate repositories because all the translations will live in this repository.

Reply:
@MikeMcQuaid yes, I mean a different script. For example, there might be additional label in translated articles with the commit hash for the English version they correspond to. So that if script notices that there were some changes in the original article since that commit it might alert about translation being out-of-date.
This would be needed for translations being part of the upstream. Even further, it would be possible to enable some specific block of html in template saying that the translation is done for some of previous versions of the original article and might be out-of-date at the moment.

Reply:
Yeh, such a script would be cool but I don't have the time to work on it myself I'm afraid. In my experience with Homebrew's homepage people will notice outdated translations and submit updates as PRs. Thanks for the thoughts, though! ❤️

Reply:
I'll try to do a PR with such script proposal if I'll have some spare time ;)